hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/TestOzoneAcls.java

```diff
@@ -103,8 +103,8 @@ public void testAclParse() {
     testMatrix.put(" world::rw", Boolean.TRUE);
     testMatrix.put(" world::a", Boolean.TRUE);
 
-    testMatrix.put(" world:bilbo:w", Boolean.FALSE);
-    testMatrix.put(" world:bilbo:rw", Boolean.FALSE);
+    testMatrix.put(" world:bilbo:w", Boolean.TRUE);
+    testMatrix.put(" world:bilbo:rw", Boolean.TRUE);
```

Comment:
This is a good catch. I think we should move to more explicit error, where if the users tries to set world and a user name, we should throw an error, that way user is not making a mistake, which we are silently ignoring.

Reply:
done.